### ISSUE TITLE
Hyper-V VM Driver Broken In 0.25.1

### ISSUE THREAD

**Issue by brockwood** [kubernetes/minikube#2630]
**BUG REPORT**

Please provide the following details:

**Environment**:

**Minikube version**: v0.25.1
- **OS**: Windows 10 Version 1709
- **VM Driver**: Hyper-V
- **ISO version**: N/A
- **Install tools**: N/A
- **Others**: N/A

**What happened**: When attempting to start minikube with `minikube start --vm-driver=hyperv`, I receive the following error:

```
Downloading Minikube ISO
 142.22 MB / 142.22 MB [============================================] 100.00% 0s
Starting VM...
E0322 09:17:52.124681    9596 start.go:159] Error starting host: Error creating host: Error executing step: Running precreate checks.
: Hyper-V PowerShell Module is not available.

 Retrying.
E0322 09:17:52.127183    9596 start.go:165] Error starting host:  Error creating host: Error executing step: Running precreate checks.
: Hyper-V PowerShell Module is not available
```

**What you expected to happen**: The start command should create the minikube VM.

**How to reproduce it** (as minimally and precisely as possible): 
- Install the Windows 10 Fall Creators Update (Version 1709)
- Ensure that the Hyper-V Windows add-on is installed on WIndows
- Run `minikube start --vm-driver=hyperv`

**Output of `minikube logs` (if applicable)**: N/A

**Anything else do we need to know**: Reverting back to 0.25.0 works as expected so it isn't a show stopper.

**Comment by dlorenc** [comment]
It looks like we unintentionally picked up some hyper-v driver changes in this release: https://github.com/kubernetes/minikube/compare/v0.25.0...v0.25.1#diff-39ccbdb27b7e34fccf018bc5200f9b5a

And unfortunately it looks like this was broken at HEAD briefly when we picked it up. The fix is in
https://github.com/docker/machine/commit/5163583eef63bfb6e55b75ceac5f1b16cbf8f201

**Comment by dlorenc** [comment]
We'll have to cut a v0.25.2 release with that vendor change applied.

**Comment by Voronenko** [comment]
I confirm issue. Looking at code: powershell.go in recent minikube

```
func hypervAvailable() error {
    stdout, err := cmdOut("@(Get-Command hyper-v\\Get-VM).ModuleName")
    if err != nil {
        return err
    }

    resp := parseLines(stdout)
    if resp[0] != "Hyper-V" {
        return ErrNotInstalled
    }

    return nil
}
```
and simple check on my box gives
```
C:\Users\slavko> (Get-Command hyper-v\Get-VM).ModuleName
hyper-v
```
So that check is quite silly, as that check in Go is case sensitive.

**Comment by Voronenko** [comment]
@dlorenc Let's choose better fix than  https://github.com/docker/machine/commit/5163583eef63bfb6e55b75ceac5f1b16cbf8f201  ?

I bet go has case-insensitive string compare.

**Comment by r2d4** [comment]
This should be fixed in 0.25.2. We can revisit vendoring a better fix upstream in docker-machine but this should unblock users upgrading.

**Comment by felixdhdez** [comment]
I had the same error with v 0.26, I had return tu v. 0.25 and its works
